dragontts/.env.example

```
@@ -0,0 +1,31 @@
+ELEVENLABS_BASE_URL=https://api.elevenlabs.io
+ELEVENLABS_INDIAN_RESIDENCY_API_KEY=
+ELEVENLABS_INDIAN_RESIDENCY_BASE_URL=
+GOOGLE_CREDENTIALS_JSON=
+GOOGLE_CREDENTIALS_PATH=
+
+# --- Storage (point both at a persistent volume in k8s) ---
+DB_PATH=data/dragontts.db
+BLOB_DIR=data/blobs
+
+# --- Cache policy ---
+MAX_CACHE_BYTES=0        # 0 = unlimited
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)